ui/lib/pki/addon/templates/keys/index.hbs

```diff
@@ -0,0 +1,63 @@
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @params={{array "keys.import"}} @type="download">
+      Import
+    </ToolbarLink>
+    <ToolbarLink @params={{array "keys.generate"}} @type="add">
+      Generate
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+<p class="has-padding">Below is information about the private keys used by the issuers to sign certificates. While
+  certificates represent a public assertion of an identity, private keys represent the private part of that identity, a
+  secret used to prove who they are and who they trust.</p>
+{{#if (gt this.model.length 0)}}
+  {{#each this.model as |model|}}
+    <LinkedBlock class="list-item-row" @params={{array "roles.role.details" model.id}} @linkPrefix={{this.mountPoint}}>
+      <div class="level is-mobile">
+        <div class="level-left">
+          <div>
+            <Icon @name="certificate" class="has-text-grey-light" />
+            <span class="has-text-weight-semibold is-underline">
+              {{model.id}}
+            </span>
+            <div class="is-flex-row has-left-margin-l has-top-margin-xs">
+              {{#if model.isDefault}}
+                <span class="tag has-text-grey-dark">default issuer</span>
```

Comment:
should this read `default key`?

Reply:
Good catch. I believe so, I'll also confirm with design. She actually didn't have this param on the designs, she had keyName, which is not available to us on the LIST endpoint.

Reply:
I'll go ahead and remove this until I get the response from the backend and design. It could be the default issuer because I don't know of a parma where is_default relates to a key. So maybe it's a key associated with the default issuer. Either way, it should be removed until clarified.

Reply:
Gotcha - I had interpreted this bit from the docs as the default key name was `default`
> [key_name](https://www.vaultproject.io/api-docs/secret/pki#key_name) (string: "") - When a new key is created with this request, optionally specifies the name for this. The global ref `default` may not be used as a name.